modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -130,17 +130,25 @@ Do not apply any expiration rules. Delete all versions after 30 days.
 'versions_retention_obligation' => '30, 30',
 ----
 
+== Restoring of a Version
+
+In the WebUI in the details view, there is a restore button next to the versions. It can be used to restore file to its past version. Restore does not delete old version, instead newer version is created.
```

Comment:
We might need to add that the retention policy is applied after restoring the file, and the retention policy might remove the old file. If not commented, the user could see the restored file deleted on some cases, and he could think it's a bug.